myhdl/conversion/_verify.py

```diff
@@ -69,7 +69,7 @@ def registerSimulator(name=None, hdl=None, analyze=None, elaborate=None, simulat
 registerSimulator(
     name="vcom",
     hdl="VHDL",
-    analyze="vcom -work work_vcom pck_myhdl_%(version)s.vhd %(topname)s.vhd",
+    analyze="vcom -2008 -work work_vcom pck_myhdl_%(version)s.vhd %(topname)s.vhd",
```

Comment:
This is a pretty big change which would affect all conversion tests. Should we document it first?

Reply:
@jck I believe I added this because there were other changes or discussion that eluded.  We can make this a separate pull-request if that simplify things.